packages/web3-utils/package.json

```diff
@@ -16,9 +16,8 @@
         "bn.js": "4.11.8",
         "eth-lib": "0.2.7",
         "ethereum-bloom-filters": "^1.0.6",
-        "ethjs-unit": "0.1.6",
-        "number-to-bn": "1.7.0",
         "randombytes": "^2.1.0",
+        "strip-hex-prefix": "1.0.0",
```

Comment:
This is probably causing a dependency duplication in another packages, as IIRC `strip-hex-prefix` is a dependency of `ethereumjs-util`, which is inderectly used by web3.

Reply:
Looked into further - the weight of this package (which uses it's own tiny dependency, [is-hex-prefix][3]) is < [20 lines][1], un-minified. It has a single published [version][2] from 3 years ago.

It looks pinned to 1.0.0 all the way through the tree:
+ ethjs-util pins both deps [here][4].
+ ethereumjs-utils pins ethjs-util [here][5]

This stuff is small and could easily be internalized somewhere although the benefit might be hard to measure. 

SilentCicero continues to actively develop eth stuff  but is working on [optimistic rollup sidechains][7]...

That's everything I found out :) and am persuadable on changing this, what's your assessment?

[1]: https://github.com/SilentCicero/strip-hex-prefix/blob/94058ad57053b4be0c7946b1abcbef0ae6c34cc8/src/index.js#L8-L14
[2]: https://www.npmjs.com/package/strip-hex-prefix
[3]: https://github.com/SilentCicero/is-hex-prefixed/blob/master/src/index.js#L7-L12
[4]: https://github.com/ethjs/ethjs-util/blob/master/package.json#L106-L109
[5]: https://github.com/ethereumjs/ethereumjs-util/blob/93544654c1b38775c2e87c8e8d6e66bf4ef7b8b3/package.json#L94
[7]: https://medium.com/@fuellabs/fuel-scaling-stablecoin-payments-10x-today-50x-in-the-near-future-961e3113b39b

Reply:
Ah! Actually this is a real problem. I think the reason the bundle isn't reducing in size is because ethereumjs-util still has the old bn.js via its pin....